.github/workflows/deploy-frontend-pr-previews.yml

```diff
@@ -5,6 +5,10 @@ on:
     branches:
       - staging
 
+env:
+  REGISTRY_URL: gcr.io
+  PROJECT_ID: airqo-250220
+
```

Comment:
Adding image registry uri global variables

Reply:
@Baalmart, here are the latest changes

Reply:
Awesome, thanks @123MwanjeMike !